projects/ngx-datatable/src/lib/themes/bootstrap.scss

```diff
@@ -1,6 +1,11 @@
-/*
-bootstrap table theme
-*/
+$datatable-ghost-cell-container-background: #fff !default;
+
+$datatable-ghost-cell-strip-background: #dee2e5 !default;
+$datatable-ghost-cell-strip-background-image: linear-gradient(to right, $datatable-ghost-cell-strip-background 0%, $datatable-ghost-cell-strip-background 10%, #fff, transparent) !default;
+$datatable-ghost-cell-strip-radius: 0 !default;
```

Comment:
Why do you use hardcoded variables instead of the bootstrap variables?

Reply:
As discussed, bootstrap variables are not defined in ngx-datatable library hence it throws error we use them here.